README.md

```diff
@@ -1,80 +1,51 @@
-    | --------------- | ---------------------------------------- |
-    | az              | For actions related to Azure             |
-    | code            | For actions related to building code     |
-    | certs           | For actions related to certificates      |
-    | db              | For actions related to databases         |
-    | git             | For actions related to Git               |
-    | iis             | For actions related to IIS               |
-    | microsoft-teams | For actions related to Microsoft Teams   |
-    | svc             | For actions related to Windows Services  |
-    | jira            | For actions related to Jira              |
-    | meta            | For actions related to running workflows |
-    | pagerduty       | For actions related to PagerDuty         |
-    | test            | For actions related to testing           |
-    | tf              | For actions related to Terraform         |
-  - [ ] Add any additional topics for an action if they apply    
     
 
 ## Inputs
-| Parameter | Is Required | Description           |
-| --------- | ----------- | --------------------- |
-| `input`   | true        | Description goes here |
-
-## Outputs
-| Output   | Description           |
-| -------- | --------------------- |
-| `output` | Description goes here |
+| Parameter                          | Is Required | Default | Description                                                                                                                                                          |
+| ---------------------------------- | ----------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `domain-name`                      | true        | N/A     | The domain name for Jira.                                                                                                                                            |
+| `proposed-build-unique-identifier` | true        | N/A     | A string that will be compared against the proposed build field in order to identify the deployment ticket. This can be the whole value or just a unique part of it. |
+| `projects-to-filter-by`            | false       | N/A     | A comma separated string of project names to filter tickets by.                                                                                                      |
+| `skip-qa-approval`                 | false       | false   | A flag determining whether or not to skip checking for QA approval. Valid values are "true" and "false".                                                             |
+| `check-subtasks`                   | false       | false   | A flag determining whether or not to check for deployment sub-tasks. Valid values are "true" and "false".                                                            |
```

Comment:
I wonder if tweaking the text or the name here might be helpful, I didn't really understand what this was for until I got to the implementation.   If you named it the same thing as our script `deploymentTaskCanBeSubtask` that would clarify what this was for.

Reply:
I ended up calling it `check-parent-task`. I'm now trying to lean into the idea that you're searching for a task to check in this action. With that in mind, hopefully this property makes more sense.